index.html

```diff
@@ -407,13 +728,20 @@ <h4>
       </ndef-record>
     </p>
     </section>
+
     <section>
     <h4>
       External type records (TNF 4)
     </h4>
     <p>
-      The NFC Forum <dfn data-no-export="">external type record</dfn>s are for client specified
-      data and must have a type name following the [[[NFC-RTD]]] standard.
+      The NFC Forum <dfn data-no-export="">external type record</dfn>s are for
+      application specified data types and are defined in [[[NFC-RTD]]].
+    </p>
+    <p>
+      The external type is a URN with the prefix `"urn:nfc:ext:"` followed by
```

Comment:
Mentioning URN is not recommended in W3C specs and the urn is all fictional as well as that is never written anywhere!

Reply:
I think we should really just avoid mentioning the URN and just state how it is written

Reply:
This comes from the NDEF spec, and they use URN. This is an informative section that explains how NDEF is supposed to work.

Reply:
Still there is specs docs from W3C saying to avoid due to a multitude of issues, and all this is useless because it's not even written as a URN...

Reply:
I thought this was important to understanding how external vs global and local types work. If I remove that text, it remains vaguely explained. With that thinking we could get rid of the whole informative explanation section, as developers can read the NDEF specs themselves.

Reply:
They most likely won't read that or have access. I just think we can explain it without referring to URNs at all, but leave it for now

Reply:
Let's sleep on this one. I don't see much harm in exposing the NDEF logic for information, but let me think it over, whether talking about URN (which I agree makes little sense in the NDEF spec to start with) would confuse any wandering soul. It would be pity to lose anyone to the dark side of the Web :).

Reply:
Kenneth, the URN's are mentioned (by you, AFAICT) in other places in the spec, too.
I suggest we merge this like this and take up the issue of URNs separately.